TemporaryProjects.15.0/StartPageExtenderPackage.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Threading;
+using System.Runtime.InteropServices;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.Shell;
+using Task = System.Threading.Tasks.Task;
+
+namespace TemporaryProjects
+{
+    [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
+    [InstalledProductRegistration("#110", "#112", "1.0", IconResourceID = 400)] // Info on this package for Help/About
+    [Guid(PackageGuids.startPageExtenderPackage15String)]
+    [SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1650:ElementDocumentationMustBeSpelledCorrectly", Justification = "pkgdef, VS and vsixmanifest are valid VS terms")]
+    public sealed class StartPageExtenderPackage : AsyncPackage
```

Comment:
Note, this doesn't have a `ProvideAutoLoad` attribute and is instead loaded by the Visual Studio 2015 compatible version of `StartPageExtenderPackage` when running inside Visual Studio 2017.

Reply:
It appears this is no longer true after 8e79128899b821efb460ccb3b795856d328a88a4